src/components/MasterDetails.js

```diff
@@ -207,12 +222,14 @@ export default class MasterDetails extends React.Component<MasterDetailsDefaultP
                 selection={this.state.selectedRows}
                 multiSelect={this.props.multiSelect}
                 noEmptySelection={this.props.noEmptySelection}
+                selectedClassName={this.props.selectedClassName}
+                tableClassName={this.props.tableClassName}
               />
               {this.props.footer}
             </div>
           </Col>
           <Col lg={detailsWidth} md={detailsWidth} sm={12} xs={12} style={{ paddingLeft: halfPadding, paddingRight: 0 }}>
-            <Detail data={detailsRow} />
+            <Detail data={detailsRow} {...this.props.detailsProps} />
```

Comment:
@luigivaldez ,
one more sort of unexpected change...
The `Details` component in the connector admin ui needs more properties than just `data` in order to function.
My understanding is that the original intention would be to pass `details` as a function taking props as a parameter.
e.g.
```jsx
          <MasterDetails
            columns={this.getTableColumns()}
            rows={rows}
            details={(props) => {
              return (
                <InspectorDetails
                  connector={props.data}
                  scannerTypes={scannerTypes}
                  isAdmin={isAdmin}
                  editConnector={editConnector}
                  openPrompt={openPrompt}
                  preview={preview}
                />
              );
            }}
```

This did work, but the problem was that the `InspectorDetails` component didn't seem to be properly maintained in the virtual DOM. Whenever a render happened, the `Details` component wouldn't just re-render, it's whole constructor would be called again which led to deeper functional issues.
This `detailsProps` property was my idea to fix the issue. If you think there is a better way, or if I'm doing something wrong, just let me know.